test/ui/fixtures/console.test.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+
+import { it } from "vitest";
+import { prettyDOM } from "@testing-library/dom"
+
+// https://github.com/vitest-dev/vitest/issues/2765
+it('regexp', () => {
+  console.log(/(?<char>\w)/)
+})
+
+// https://github.com/vitest-dev/vitest/issues/3934
+it('html-raw', async () => {
+  console.log(`
+<form>
+  <label for="email">Email Address</label>
+  <input name="email" />
+  <button>Submit</button>
+</form>
+`);
+})
+
+// https://github.com/vitest-dev/vitest/issues/1279
+it('html-pretty', () => {
+  const div = document.createElement("div");
+  div.innerHTML = `
+    <form>
+      <label for="email">Email Address</label>
+      <input name="email" />
+      <button>Submit</button>
+    </form>
+  `.replaceAll(/\n */gm, ""); // strip new liens
+  console.log(prettyDOM(div))
+})
```

Comment:
I add e2e tests https://github.com/vitest-dev/vitest/pull/4724/commits/64ae9d2e4538fd0d1deb979dd6dd4827a789fe31. Asserting dom format was difficult, so the actual testing is limited to only checking `/(?<char>\w)` string.